src/RoadRegistry.BackOffice.ExtractHost/RoadRegistry.BackOffice.ExtractHost.csproj

```diff
@@ -24,7 +24,10 @@
     <Content Include="Dockerfile" CopyToOutputDirectory="Always" />
     <Content Include="init.sh" CopyToOutputDirectory="Always" />
     <Content Include="appsettings.json" CopyToOutputDirectory="Always" />
-    <Content Include="appsettings.*.json" CopyToOutputDirectory="Always" />
+    <Content Include="appsettings.*.json">
```

Comment:
remove this block (i.e. restore to what it was)

Reply:
Done